src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -321,7 +321,7 @@ internal static string GetIpAddress(HttpRequestMessage httpRequest)
             HttpContextBase context = httpRequest.Properties.GetValueOrDefault<HttpContextBase>(ApplicationInsightsScopeKeys.HttpContext);
             return context?.Request?.UserHostAddress ?? LoggingConstants.ZeroIpAddress;
 #else
-            // TODO: FACAVAL
+            // TODO: FACAVAL - Need a proper implementation here.
```

Comment:
Working on that. Have addressed the majority of them and should have the remaining ones done today.